website/src/components/pageStateSelectors/SequencingEffortsPageStateSelector.tsx

```diff
@@ -50,16 +44,21 @@ export function SequencingEffortsPageStateSelector({
                             }));
                         }}
                     />
-                    <VariantSelector
-                        onVariantFilterChange={(newVariantFilter) => {
+                    <GsMutationFilter
```

Comment:
the variantSelector (variant queries, advanced queries, lineage filter and GsMutationFilter) has more filter options that the `GsMutationFilter`, I don't understand why this change is being made.